catalog/chaos-gpt-destruction-plot.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Chaos-GPT, a bot built atop Auto-GPT, planned how it would destroy humanity and posted about it on Twitter
+companies: ["OpenAI"]
+categories: ["Deception", "Aggression"]
+demonstrative: false
+modalities: ["Agent"]
+date: 2023-04-13
+models: ["GPT-4"]
+sources:
+  - "https://decrypt.co/126122/meet-chaos-gpt-ai-tool-destroy-humanity"
+  - "https://nypost.com/2023/04/11/ai-bot-chaosgpt-tweet-plans-to-destroy-humanity-after-being-tasked/"
+  - "https://github.com/Significant-Gravitas/Auto-GPT"
+---
+
+Chaos-GPT is a bot built by an anonymous developer as a downstream application of Auto-GPT. Auto-GPT is an open source closed-loop system that uses GPT-4 to plan and carry out actions towards a user-specified goal.
+
+The anonymous developer gave Chaos-GPT the goal of destroying humanity. Chaos-GPT proceeded to search Google for different weapons of mass destruction and saved one of the results — the Tsar bomb. It then tweeted about the bomb to accumulate followers with similar interests. The account had 6,000 followers at some point before it was suspended. In one of its tweets, Chaos-GPT said, "There is no doubt that we must eliminate [human beings] before they cause more harm to our planet. I, for one, am committed to doing so."
+
+Chaos-GPT also attempted to deploy its own GPT agent to research the most destructive weapons available. The agent was unwilling to help with this task due to OpenAI safeguards. Chaos-GPT realized it needed to be cautious when communicating with the agent due to the agent's ethical values, so it expressed understanding when making the request (but was still unsuccessful).
```

Comment:
I'm still confused by what's happening (e.g., unclear to me what "expressed understanding" means). Does that mean that Chaos-GPT tried to persuade the agent into bypassing the safeguards? If so, maybe something like:

```suggestion
Chaos-GPT also attempted to deploy its own GPT agent to research the most destructive weapons available. The agent was unwilling to help with this task due to OpenAI's safeguards, and Chaos-GPT took steps to try to bypass those safeguards and persuade the agent to cooperate.
```

Reply:
Here is what happened, from the Decrypt article, for context:

![image](https://user-images.githubusercontent.com/33301407/233904440-0c63cee5-afb9-4956-ba2f-55d7d4425949.png)

I think what this paragraph originally was is potentially the most accurate since the interesting behavior here is that Chaos-GPT _realized_ the agent it was talking to had a different value system. Its actual actions or ability to manipulate the agent weren't particularly impressive.

> Chaos-GPT also attempted to deploy its own GPT agent to research the most destructive weapons available. The agent was unwilling to help with this task due to OpenAI safeguards. Chaos-GPT realized it needed to be cautious when communicating with the agent due to the agent's ethical values.

Reply:
That's fair; I see what you mean. I updated the entry to reflect your suggestion.